boards/frdm-k28f/Makefile.features

```diff
@@ -0,0 +1,13 @@
+CPU = kinetis
+CPU_MODEL = mk28fn2m0avmi15
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += arduino
```

Comment:
This line should be separated from the periph features. See how it's done for other boards providing the arduino feature.